catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -1098,23 +1062,36 @@ public boolean isAvailable(SourceMonitor callback) {
    */
   private class WfsSourceAvailabilityCommand implements AvailabilityCommand {
 
+    private void availabilityChanged(boolean isAvailable) {
```

Comment:
I was just reading the book `Refactoring` where Martin Fowler talks about breaking methods like this (ones that take a boolean) and splitting them into two methods. Then have the calling code decide which is the correct method to call.